File: speech-to-text/request.js
const https = require('https')
const request = require('request');
const fs = require('fs');
require('dotenv').config();

const WIT_AI_TOKEN = process.env.WIT_AI_TOKEN;

request({
    url: 'https://api.wit.ai/speech?v=20200513',
    method: 'POST',
    headers: {
      'Content-type' : 'audio/wav',
      'Authorization' : `Bearer ${ WIT_AI_TOKEN }`
    },
    encoding: null,
    body: fs.createReadStream('./speech-to-text/sample.wav')
    }, (error, response, body) => {
        if (error) {
            // response.json({name : error});
            console.log(response);
        } else {
            console.log(JSON.parse(response.body.toString()));
        }
    });

File: speech-to-text/request.py
import json
import sys
from wit import Wit
# from utils import uprint
access_token = ''

with open('speech-to-text/config.json') as json_file:
    data = json.load(json_file)
    access_token = data['wit_api_key']
client = Wit(access_token)

resp = None
try:
    with open('speech-to-text/sample.wav', 'rb') as f:
        resp = client.speech(f, {'Content-Type': 'audio/wav'})

    print(resp['text'].encode('utf8'))
except Exception as e:
    print(e)
sys.stdout.flush()